.craft.yml

```diff
@@ -1,7 +1,4 @@
-minVersion: '0.21.0'
-github:
-  owner: getsentry
-  repo: craft
+minVersion: '0.22.2'
```

Comment:
I think this should be `0.22.3`.

Reply:
Good guess but no 🙂 

The latest released version of Craft is `0.22.2` and this new version will be `0.23.0`. That said, even then we cannot set this to `0.23.0` as the validation of the config would fail at the `craft prepare` stage when we try to release `0.23.0`. Ironically, we use the latest master of Craft when publishing so we will be able to use this new feature _before_ it is released in `0.23.0`.

I'll follow up with a patch to bring this number to `0.23.0` after the release tho.

Reply:
In that case why bother bumping `minVersion` here at all? Is that strictly necessary?

Reply:
>In that case why bother bumping minVersion here at all? Is that strictly necessary?

Yes as I remove the top-level `github` key which forces Craft to infer it, which is only introduced in `0.22.0`.

Reply:
> which forces Craft to infer it

Sorry ... to infer what?

I guess more generally, why do we ever need to specify a `minVersion` here if we will always use the latest version anyway? Maybe I'm misunderstanding ...

Reply:
See #208 re "infer what".

>why do we ever need to specify a minVersion here if we will always use the latest version anyway? Maybe I'm misunderstanding

We _hope_ to use the latest version at all times but we don't. `minVersion` ensures capabilities and sometimes config stability as in the case of #202.